--- Vc-1.3.0/CMakeLists.txt.0000~	2016-12-16 11:46:34.178663998 +0100
+++ Vc-1.3.0/CMakeLists.txt	2016-12-16 11:47:17.737632408 +0100
@@ -134,7 +134,7 @@ if(MIC_FOUND)
 endif()
 
 set(_srcs src/const.cpp)
-if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "([x3-7]86|AMD64)")
+if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "(i.86|[x3-7]86|AMD64)")
 
    list(APPEND _srcs src/cpuid.cpp src/support_x86.cpp)
    vc_compile_for_all_implementations(_srcs src/trigonometric.cpp ONLY SSE2 SSE3 SSSE3 SSE4_1 AVX SSE+XOP+FMA4 AVX+XOP+FMA4 AVX+XOP+FMA AVX+FMA)# AVX2+FMA+BMI2)
